storybook/storybook-react/src/OrderedList/OrderedList.stories.tsx

```diff
@@ -104,3 +104,16 @@ export const WithoutMarkers: Story = {
     markers: false,
   },
 }
+
+export const InverseColor: Story = {
+  args: {
+    inverseColor: true,
```

Comment:
We should add the `inverseColor` boolean to the `argTypes` as well, the control currently isn't visible for all stories. Same for the Unordered list stories.

Reply:
Uh-oh, don’t let @dlnr read this 🫣. He and I decided earlier today to move the arg and decorator over from the other stories, because a nested list isn’t affected by the arg and would appear incorrectly.

Reply:
So do I add the `argType` that break the stories or leave it to the story to illustrate how the prop works?

Reply:
Preferably neither ;) You can pass the args to the children of a story with the render function, right? So you can pass the value of `inverseColor` to the nested list?

Reply:
Done!